pkg/svcat/service-catalog/broker.go

```diff
@@ -88,7 +92,7 @@ func (sdk *SDK) RetrieveBrokers(opts ScopeOptions) ([]Broker, error) {
 func (sdk *SDK) RetrieveBroker(name string) (*v1beta1.ClusterServiceBroker, error) {
 	broker, err := sdk.ServiceCatalog().ClusterServiceBrokers().Get(name, v1.GetOptions{})
 	if err != nil {
-		return nil, fmt.Errorf("unable to get broker '%s' (%s)", name, err)
+		return nil, errors.Wrapf(err, "unable to get broker '%s' (%s)", name, err)
```

Comment:
when using errors.Wrapf, the error message is concatenated with your format text, so you don't need the final (%s), and err param - - the text from the original error is automatically included and using what you have in here now will duplicate the error text.  I used this and Carolyn pointed out this issue.

Reply:
Fixed this